### PR TITLE
Bug 2009352: IR-254: Collecting registry and image stream usage

### PR DESCRIPTION
**Pull request by ricardomaraschini**
As an effort to evaluate how often customers are using Image Streams and
the internal registry this commit introduces two new metrics to be
collected:

- The amount of Image Stream Tags in the cluster
- The amount of pushes and pulls executed against the registry

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: Documentation/data-collection.md [hunk 1]
```diff
@@ -648,6 +648,16 @@ data:
     #
     # cluster:velero_restore_total:max measures the current number of existent restores
     - '{__name__="cluster:velero_restore_total:max"}'
+    #
+    # owners: (@openshift/image-registry)
```
Comment (dmage) [inline Documentation/data-collection.md:652]: If it's a GitHub team, it should be @openshift/openshift-team-image-registry.

Comment (ricardomaraschini) [reply Documentation/data-collection.md]: fixed

**Comment by ricardomaraschini** [comment]
@simonpasquier rebased.

**Comment by jan--f** [comment]
ty
/lgtm
/unhold

**Pull request merged**